.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,45 @@
+name: New feature proposal
+description: Propose a new component, pattern, style or documentation to add to the ONS Design System
+title: "[Feature]: "
+labels: ["New feature proposal", "triage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before you fill in this proposal template:**
+        - If you are suggesting a change to a component, pattern or style that is already in the ONS Design System, please comment on its GitHub discussion instead (https://github.com/ONSdigital/design-system/discussions). You can get to the discussion using the direct link at the bottom of each page in the Design System.
```

Will the template let us hyperlink and hide the url?
```suggestion
        - If you are suggesting a change to a component, pattern or foundation that is already in the ONS Design System, please comment on its [GitHub discussion instead] (https://github.com/ONSdigital/design-system/discussions). You can get to the discussion using the direct link at the bottom of each page in the Design System.
```